CODEOWNERS

```diff
@@ -193,18 +179,83 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
+# compression code
+/*/extensions/filters/http/decompressor @rojkov @kbaichoo
+/*/extensions/filters/http/gzip @gsagula @kbaichoo
+/*/extensions/filters/http/common/compressor @gsagula @rojkov @kbaichoo
+/*/extensions/filters/http/compressor @rojkov @kbaichoo
+/*/extensions/compression/brotli @rojkov @kbaichoo
+/*/extensions/compression/common @kbaichoo @rojkov
+/*/extensions/compression/gzip/decompressor @rojkov @kbaichoo
+/*/extensions/compression/gzip/compressor @rojkov @kbaichoo
+/*/extensions/compression/gzip/common @rojkov @kbaichoo
+# cel
+/*/extensions/access_loggers/filters/cel @kyessenov @douglas-reid @adisuissa
+# health cehck
+/*/extensions/filters/http/health_check @mattklein123 @adisuissa
+
+# "Underowned" extensions (no backup reviewers, please volunter!)
+# Stateful session
+/*/extensions/http/stateful_session/cookie @wbpcode @UNOWNED
+/*/extensions/filters/http/stateful_session @wbpcode @UNOWNED
+# tap
+/*/extensions/filters/http/tap @mattklein123 @UNOWNED
+/*/extensions/common/tap @mattklein123 @UNOWNED
+# rate limit
+/*/extensions/filters/network/local_ratelimit @mattklein123 @UNOWNED
+# tap
+/*/extensions/transport_sockets/tap @mattklein123 @UNOWNED
+# lua
+/*/extensions/filters/http/lua @mattklein123 @UNOWNED
+/*/extensions/filters/common/lua @mattklein123 @UNOWNED
+# stats
+/*/extensions/stat_sinks/statsd @mattklein123 @UNOWNED
+/*/extensions/stat_sinks/common @mattklein123 @UNOWNED
+/*/extensions/stat_sinks/common/statsd @mattklein123 @UNOWNED
+# proxy protocol
+/*/extensions/filters/listener/proxy_protocol @ggreenway @UNOWNED
+
+# "Unowned" extensions (currently without 1 maintainer or 2 owners)
+/*/extensions/formatter/req_without_query @UNOWNED @tsaarni
+# ext_authz
+/*/extensions/filters/common/ext_authz @esmet @gsagula @UNOWNED
+/*/extensions/filters/http/ext_authz @esmet @gsagula @UNOWNED
+/*/extensions/filters/network/ext_authz @esmet @gsagula @UNOWNED
+# rbac
+/*/extensions/filters/network/rbac @yangminzhu @UNOWNED
+/*/extensions/filters/http/rbac @yangminzhu @UNOWNED
+/*/extensions/filters/common/rbac @yangminzhu @UNOWNED
+/*/extensions/filters/http/ip_tagging @UNOWNED @UNOWNED
+/*/extensions/filters/http/cors @UNOWNED @UNOWNED
+/*/extensions/filters/listener/original_dst @UNOWNED @UNOWNED
+/*/extensions/access_loggers/grpc @UNOWNED @UNOWNED
+/*/extensions/access_loggers/file @UNOWNED @UNOWNED
+/*/extensions/tracers/zipkin @UNOWNED @UNOWNED
+/*/extensions/tracers/dynamic_ot @UNOWNED @UNOWNED
+/*/extensions/tracers/opencensus @UNOWNED @UNOWNED
+/*/extensions/tracers/common @UNOWNED @UNOWNED
+/*/extensions/tracers/common/ot @UNOWNED @UNOWNED
+/*/extensions/retry/host/previous_hosts @UNOWNED @UNOWNED
+/*/extensions/filters/network/mongo_proxy @UNOWNED @UNOWNED
+
+# Intentionally exempt (treated as core code)
+/*/extensions/filters/common @UNOWNED @UNOWNED
+/*/extensions/filters/http/common @UNOWNED @UNOWNED
+/*/extensions/filters/network/common @UNOWNED @UNOWNED
```

Comment:
I want to pick some as backup if possible. Seems that first-pass reviewers do not cover specified fields, I will help review anyway.